### DISCUSSION TITLE
Increments

### DISCUSSION THREAD

**Discussion by jorgepastrana1980** [rsjaffe/MIDI2LR#704] [Q&A]
hi! i have been searching with no luck, is there a way to set the fader on the xtouch mini to make exposure increments of 0.10+-?

**Comment by illuminancephoto** [comment]
On an X-Touch Compact I set the limit for Exposure to -5.0 through +5.0, right click the LR Command to get the "Adjust CC dialog" and set an absolute range of 0-100, then set the X-Touch to only send a range of 35-75.  This gets me a fader that moves between -1.50 and +2.50 with 0.10 increments.  You could have the fader send the whole 1-100 range, but the movements to get a 0.10 change are tiny and hard to control.
